rasa/importers/importer.py

```diff
@@ -0,0 +1,274 @@
+import asyncio
+from functools import reduce
+from typing import Text, Optional, List, Dict
+import logging
+
+import rasa.utils.common
```

Comment:
can be removed as you have further down `import rasa.utils.common as common_utils`